### DISCUSSION TITLE
Template only in BS5 for Joomla 4

### DISCUSSION THREAD

**Discussion by Magnytu2** [joomla/joomla-cms#32360] [2. Cold Features]
Hello, I am looking to make a template for the Joomla community. Only in bootstrap 5 and in the format of the examples on the bootstrap site.
I'm looking for documentation to make this template with Joomla 4, but also so that there is an admin part that manages the colors and layout format in the three areas independently (header, content, bottom).

![Screenshot_2021-02-08 Blog Template · Bootstrap v5 0](https://user-images.githubusercontent.com/26197929/107262675-23255600-6a41-11eb-9a9a-7ed1412b80f7.png)

**Comment by drmenzelit** [comment]
I moved this to discussions, because it is not an issue of Joomla.

You can use Cassiopeia as example on how to build a template for Joomla 4. You can also have a look at Protostar for the use of parameters for colours used in the template. In Protostar inline CSS is used. I don't think that is a good way, but if you want to use the parameters inside a CSS file, you will need some programming. Maybe some template expert can help you with this.

**Reply by richard67** [reply]
@Magnytu2 Does @drmenzelit 's comment above answer your question?

**Reply by Magnytu2** [reply]
No, because Cassiopeia is a real technical break with the CSS Grid which is very complicated to understand. I cannot find support to learn and be able to apply it on Joomla! 4. That's why I wanted to start from scratch with Bootstrap 5, with a build as described in Bootstrap documentation. But I realize that menu management is also complex.
As for the admin part, which I want very accessible for all, ditto, I searched a lot, but I can not find much documentation.
